tastypie/resources.py

```diff
@@ -10,7 +10,14 @@
 import django
 from django.conf import settings
 from django.conf.urls import patterns, url
-from django.contrib.gis.db.models.fields import GeometryField
+
+# Django 1.8 will run without GEOS so fail silently if not installed
+try:
+    from django.contrib.gis.db.models.fields import GeometryField
+    use_geos = True
+except:
```

Comment:
Could we except only on the specific error that's raised when GEOS isn't installed?